processor/datadogprocessor/README.md

```diff
@@ -0,0 +1,27 @@
+# Datadog Processor
+
+| Status                   |               |
+|--------------------------|---------------|
+| Stability                | [beta] |
+| Supported pipeline types | traces        |
+| Distributions            | [contrib]     |
+
+## Description
+
```

Comment:
> We should explain what the level of support for this processor if someone decides to use this without the Datadog exporter.

I feel that I've addressed that in the Configuration part. But please feel free to suggest rephrasing / restructuring. I am open to changes.

> I would also use 'Datadog APM Stats' throughout the README to make it clear this is a Datadog-specific concept.

Done 👍🏻

Reply:
Sorry, I should have said 'without using Datadog as a backend' instead of 'without the Datadog exporter'. I would say in that case we should recommend using the spanmetrics processor instead.